### DISCUSSION TITLE
Theme is not working in webpack react.

### DISCUSSION THREAD

**Discussion by bannarisoftwares** [tailwindlabs/tailwindcss#8007] [Help]
I want to switch the theme dynamically in my react project. and my react app is build and serve using webpack. so i inspired from this sample [carco sample](https://github.com/bandrewfisher/theming-react-components.git) . This repo contains the example with switching b/w two themes at run time. and this repo builded using `carco`. I tried to migrate from carco to webpack [webpack sample](https://github.com/bannarisoftwares/tailwind-theme-webpack-issue.git ). After migrating to webpack, theme switching is stoped working. While i inspect using inspect window from chrome, the codes are changing... but color are not reflecting.

So anyone please help to resolve the issue, Thanks in advance.

- [carco sample](https://github.com/bandrewfisher/theming-react-components.git) -> https://github.com/bandrewfisher/theming-react-components.git
- [webpack sample](https://github.com/bannarisoftwares/tailwind-theme-webpack-issue.git ) -> https://github.com/bannarisoftwares/tailwind-theme-webpack-issue.git

**Comment by natheeshsunway** [comment]
Seems like need to add 

```
module.exports = {
  plugins: [
    require('postcss-import'),
    require('tailwindcss'),
    require('autoprefixer'),
  ],
}
```
in `postcss.config.js`

**Reply by bannarisoftwares** [reply]
This is working for tailwind css 2.X version, not working for tailwind css 3.X version
